### DISCUSSION TITLE
[config-plugin] Mod for MainApplication.java

### DISCUSSION THREAD

**Discussion by lHersey** [expo/expo#14060] [Archived: Questions]
I'm trying to add a package to the `protected List<ReactPackage> getPackages()`

This package cannot be autolinked (https://docs.getsocial.im/guides/getting-started/react-native/) and I only found a mod for the main activity, is there a way to add a custom mod or this should be added directly to Expo?

Thanks in advance

**Comment by barthap** [comment]
Maybe https://github.com/expo/expo-cli/pull/3909 and https://github.com/expo/expo/pull/14732 will be an answer when they get merged and published
